docs/source/containerization/kubernetes.mdx

```diff
@@ -1,23 +1,23 @@
 ---
 title: Deploy in Kubernetes
-subtitle: Self-hosted deployment of Apollo Router in Kubernetes
-description: Deploy the Apollo Router in Kubernetes using Helm charts. Customize configurations, enable metrics, and choose values for migration.
+subtitle: Self-hosted deployment of GraphOS Router in Kubernetes
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)